app/src/main/res/values/strings.xml

```diff
@@ -1420,10 +1420,8 @@
     <string name="preferences_passwords_autofill">Autofill</string>
     <!-- Preference for syncing saved logins in Fenix -->
```

Comment:
```suggestion
    <!-- Preference option for syncing saved logins across devices. This is displayed when the user is signed into sync -->
```

Reply:
1. `preference` and  `preference option` are the same for android devs imo. 
2. this comment wasn't written by me
3. _"This is displayed when the user is signed into sync"_ - This is an implementation description.